libraries/chain/steem_evaluator.cpp

```diff
@@ -1291,6 +1291,31 @@ namespace golos { namespace chain {
                         "Votes are not allowed on the comment.");
                 }
 
+                std::vector<delegator_vote_interest_rate> delegator_vote_interest_rates;
+                if (_db.has_hardfork(STEEMIT_HARDFORK_0_19__756) && voter.received_vesting_shares > asset(0, VESTS_SYMBOL)) {
+                    const auto& vdo_idx = _db.get_index<vesting_delegation_index>().indices().get<by_received>();
+                    auto vdo_itr = vdo_idx.lower_bound(voter.name);
+                    for (; vdo_itr != vdo_idx.end() && vdo_itr->delegatee == voter.name; ++vdo_itr) {
+                        if (vdo_itr->interest_rate == 0) {
+                            continue;
+                        }
+
+                        delegator_vote_interest_rate dvir;
+                        dvir.account = vdo_itr->delegator;
+                        dvir.interest_rate = vdo_itr->vesting_shares.amount.value * STEEMIT_100_PERCENT * vdo_itr->interest_rate
+                            / voter.effective_vesting_shares().amount.value;
+                        delegator_vote_interest_rates.push_back(dvir);
+                    }
+                }
+                auto set_delegator_vote_interest_rates = [&](comment_vote_object& cvo) {
+                    if (_db.has_hardfork(STEEMIT_HARDFORK_0_19__756)) {
+                        cvo.delegator_vote_interest_rates.clear();
+                        for (auto& dvir : delegator_vote_interest_rates) {
```

Comment:
if `cvo.weight == 0` here, then all delegators will get 0 reward, so it's not needed to fill `delegator_vote_interest_rates`